network-api/networkapi/templates/fragments/buyersguide/product_tab_header.html

```diff
@@ -1,35 +1,46 @@
 {% load wagtailcore_tags bg_selector_tags l10n i18n localization static %}
 
-{% with tab_class="tw-bg-white tw-text-black tw-font-zilla tw-text-3xl tw-py-4 tw-px-12 tw-flex tw-justify-center tw-items-center medium:tw-flex-1 tw-shrink-0 tw-text-center tw-relative tw-cursor-pointer tw-ease-in tw-transform tw-duration-150 medium:tw-w-auto" %}
```

Comment:
Basically changed the DOM structure to something more semantic and more a11y friendly.

Before:
```
<div ...>
  <span>tab name</span>
  <span>tab name</span>
  <span>tab name</span>
</div>
```

After:
```
<ul>
  <li>
    <button>tab name</button>
  </li>
  <li>
    <button>tab name</button>
  </li>
  <li>
    <button>tab name</button>
  </li>
</ul>
```

Reply:
Other changes:

- added `role` attribute to some elements
- adjustment to class list